src/utils/ElementType.js

```diff
@@ -7,31 +7,23 @@
 import React from 'react';
 import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined';
 import DescriptionIcon from '@material-ui/icons/Description';
-import PanToolIcon from '@material-ui/icons/PanTool';
 import FilterListIcon from '@material-ui/icons/FilterList';
-import FilterIcon from '@material-ui/icons/Filter';
 
 export const elementType = {
     DIRECTORY: 'DIRECTORY',
     STUDY: 'STUDY',
     FILTER: 'FILTER',
-    SCRIPT: 'SCRIPT',
-    SCRIPT_CONTINGENCY_LIST: 'SCRIPT_CONTINGENCY_LIST',
-    FILTERS_CONTINGENCY_LIST: 'FILTERS_CONTINGENCY_LIST',
+    CONTINGENCY_LIST: 'CONTINGENCY_LIST',
 };
```

Comment:
We should add subtypes enums here too for CONTINGENCY_LIST and FILTER, then edit the getFileIcon function to accept a subtype parameter and keep our FilterIcon and PanToolIcon otherwise we will have a regression in gridExplore making it difficult to distinguish filter or script contingency lists and same with filters.

Reply:
gridExplore dont use the getFileIcon function in commons-ui : it already selects the right icons based on the elements type and subtype.
getFileIcon is only used by study-app in directory-item-selector.js file (in directory2Tree callback), from requests to the directory-server which knows only the elements type, and not their subtypes ...

Reply:
Ok then this getFileIcon logic by type (and subtypes) is duplicated, we should fix this later.
Anyway if we don't need the distinction in the AS it's ok for now.